roles/vault_pki/defaults/main.yml

```diff
@@ -23,6 +23,7 @@ vault_pki_intermediate_key_bits: 4096
 vault_pki_intermediate_roles: {}
 
 vault_pki_generate_certificates: false
+vault_pki_write_pem_bundle: false
 
 vault_pki_write_certificates_host: "localhost"
 vault_pki_write_certificates_directory: ""
```

Comment:
unrelated: This isn't referenced anywhere else, but `vault_pki_certificates_directory` is.

Reply:
Thanks. fixed.

Reply:
Done.